### PR TITLE
[carry 11669] Add Stale image info under Viewing Repostories and markdown fix-ups

### PR DESCRIPTION
**Pull request by thaJeztah**
carries https://github.com/docker/docker.github.io/pull/11669 with my review comments addressed, the commits squashed, and some markdown fix-ups I found on this page.

### CODE REVIEW

**Comment by thaJeztah** [comment]
@FranchommeE  @spencerhcheng @usha-mandya PTAL

**Comment by usha-mandya** [comment]
@thaJeztah Wondering if we can reduce the size of the images (Create repository, visibility settings, delete repository)in the Private repositories section. Looks good otherwise. Thanks.

**Comment by FranchommeE** [comment]
@thaJeztah LGTM , thanks!
I had marked my 1st PR #11669 as Draft because it was missing screenshots.
Also I am unsure of the process to be in sync with Removing the feature flag (and make the feature GA) and deploying the updated documentation. There is also a conflict to expect upon merging, as this update shouldn't override the changes brought by #11668

**Comment by thaJeztah** [comment]
> Wondering if we can reduce the size of the images (Create repository, visibility settings, delete repository)in the Private repositories section. Looks good otherwise. Thanks.

I added a "max-width" to the images (included in https://github.com/docker/docker.github.io/pull/11686 as well). We should probably have a look at defining proper styles for those (using the bootstrap/css grid), also w.r.t. "hidpi" images. 


> There is also a conflict to expect upon merging, as this update shouldn't override the changes brought by #11668

Ah, yes, thanks for the heads-up; merge conflict should be trivial to fix

File: docker-hub/repos.md [hunk 1]
```diff
@@ -128,6 +128,23 @@ To view individual tags, click on the **Tags** tab.
...
+than 1 month, i.e.:
+
+* It has not been pulled for more than 1 month
+* And it has not been pushed for more than 1 month
+
+A multi-architecture image is considered stale if all single-architecture images
+part of its manifest are stale.
+
+To delete a tag, select the corresponding checkbox and select the action "Delete"
+above.
```
Comment (spencerhcheng) [inline docker-hub/repos.md:141]:
```suggestion
To delete a tag, select the corresponding checkbox and select the action **Delete**.
```

**Review by spencerhcheng — APPROVED** [review]

File: docker-hub/repos.md (hunk 1 above)
Comment (thaJeztah) [reply docker-hub/repos.md:141]: update, thanks 👍

**Comment by spencerhcheng** [comment]
I'm creating a PR to remove the feature flags: https://docker.atlassian.net/browse/HUB-4619 which will be ready to deploy any time. I'll be out from Nov 5th to the 11th - giving @ddebroy a heads up on deployment coordination.

File: docker-hub/repos.md (hunk 1 above)
Comment (spencerhcheng) [reply docker-hub/repos.md]: Can we also remove `above.` and maybe switch the wording to:

`To delete a tag, select the corresponding checkbox and select the  **Delete** action.`

**Review by spencerhcheng — DISMISSED** [review]

File: docker-hub/repos.md [hunk 2]
```diff
@@ -128,6 +128,23 @@ To view individual tags, click on the **Tags** tab.
...
+An image is considered stale if there has been no push/pull activity for more
+than 1 month, i.e.:
+
+* It has not been pulled for more than 1 month
+* And it has not been pushed for more than 1 month
+
+A multi-architecture image is considered stale if all single-architecture images
+part of its manifest are stale.
+
+To delete a tag, select the corresponding checkbox and select the action **Delete**
```
Comment (usha-mandya) [inline docker-hub/repos.md:140]:
```suggestion
To delete a tag, select the corresponding checkbox and select **Delete** from the **Action** drop-down list.
```

Comment (thaJeztah) [reply docker-hub/repos.md]: Ah, thx, yes, that reads better. Updated 👍

**Comment by usha-mandya** [comment]
@FranchommeE Could you let us know when we are planning to go live with this update? Thanks.

**Comment by FranchommeE** [comment]
@usha-mandya Perfect timing! I was just pinging you on Slack about this: we plan on going live this week. I am not sure yet of the exact date/time.

**Review by usha-mandya — APPROVED** [review]
LGTM.

**Pull request merged**